docs/diagrams/FilePathDiagrams/FilePathActivityDiagram.puml

```diff
@@ -0,0 +1,28 @@
+@startuml
+!pragma useVerticalIf on
+
+start
+:User executes SetPersonFilePathCommand;
+
+:Verify SetPersonFilePathCommand keyword;
+if() then ([else])
+:show error message;
+stop
+
+else ([is valid filePath])
+:Verify Index in SetPersonFilePathCommand;
+if() then ([else])
+:show error message;
+stop
+
+else ([valid keyword])
+:Verify filePath in SetPersonFilePathCommand;
+if() then ([else])
+:show error message;
+stop
+
+else ([is valid filePath and Index])
```

Comment:
I think the if-else branches conditions in the UML diagram might be input wrongly. 
E.g. for the first conditional branch, 

it says Verify keyword, then the conditional is valid filepath
which I think might be an error